### PR TITLE
Add doc for ADOT Collector SSM support

### PR DESCRIPTION
**Pull request by vastin**
Add doc for ADOT Collector SSM support.
The feature is implemented in following PRs.
https://github.com/aws-observability/aws-otel-collector/pull/436
https://github.com/aws-observability/aws-otel-collector/pull/449

### CODE REVIEW

File: src/docs/setup/ec2.mdx [hunk 1]
```diff
@@ -68,6 +68,33 @@ sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -a status
 <img src={img10} alt="Diagram" style="margin: 30px 0;" />
 <div style="height: 40px" />
 
+<SectionSeparator />
+
+## Install AWS Distro for OpenTelemetry Collector on EC2 instance using AWS Systems Manager Distributor
+
+To use [AWS Systems Manager Distributor](https://docs.aws.amazon.com/systems-manager/latest/userguide/distributor.html) to install AWSDistroOTel-Collector package, the EC2 instance need to meet follwoing requiements.
```
Comment (mxiamxia) [inline src/docs/setup/ec2.mdx:75]:
```suggestion
To use [AWS Systems Manager Distributor](https://docs.aws.amazon.com/systems-manager/latest/userguide/distributor.html) to install AWS OTel Collector on EC2 instances, the target EC2 instances need to meet following requirements.
```

Comment (mxiamxia) [reply src/docs/setup/ec2.mdx:75]: `AWSDistroOTel-Collector package` is an internal name for SSM to pull the AWS OTel Collector. Can we update it to `AWS OTel Collector` as suggested above?

File: src/docs/setup/ec2.mdx [hunk 2]
```diff
@@ -68,6 +68,33 @@ sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -a status
...
+
+* SSM agent version 2.3.1550.0 or later installed
+* Permissions Policy AmazonSSMManagedInstanceCore attached in IAM Role of EC2 instance
+
+To install AWSDistroOTel-Collector package using AWS Systems Manager Distributor.
+
+1. Based on your preferences, prepare AWS Distro for OpenTelemetry Collector YAML configuration file according to [AWS OTel Collector Configuration](https://github.com/aws-observability/aws-otel-collector/#aws-otel-collector-configuration).
+
+2. Copy AWS Distro for OpenTelemetry Collector YAML configuration file encoded as Base64 to a new SSM parameter in [SSM Parameter Store](https://us-west-2.console.aws.amazon.com/systems-manager/parameters/).
```
Comment (mxiamxia) [inline src/docs/setup/ec2.mdx:84]: Can we put a note here why do we need to encode plain YAML config to Base64 format?

File: src/docs/setup/ec2.mdx [hunk 3]
```diff
@@ -68,6 +68,33 @@ sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -a status
...
+    * For Linux, `base64 -w0 config.yaml > aoc-config-base64.txt`
+    * For Windows, `certutil.exe -f -encodehex config.yaml aoc-config-base64.txt 0x40000001`
+    * For MacOSX, `base64 -i config.yaml -o aoc-config-base64.txt` 
+
+3. Navigate to the [SSM Distributor](https://console.aws.amazon.com/systems-manager/distributor) console and open the Owned by Amazon tab. Select AWSDistroOTel-Collector and choose Install one time.
```
Comment (mxiamxia) [inline src/docs/setup/ec2.mdx:89]:
```suggestion
3. Navigate to the [SSM Distributor](https://console.aws.amazon.com/systems-manager/distributor) console and open the `Owned by Amazon` tab. Select AWSDistroOTel-Collector and choose `Install one time`.
```

File: src/docs/setup/ec2.mdx [hunk 4]
```diff
@@ -68,6 +68,33 @@ sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -a status
...
+
+4. Update the SSM parameter you created in the first step by replacing "Additional Arguments" with the following (e.g. SSM parameter name is "aoc-config-base64"):
+```
+{"SSM_CONFIG": "{{ssm:aoc-config-base64}}"}
+```
+
+5. Continue to select the EC2 instance in Targets and click Run button to start installation.
+
```
Comment (mxiamxia) [inline src/docs/setup/ec2.mdx:97]: how about select ec2 ASG?

File: src/docs/setup/ec2.mdx (hunk 1 above)
Comment (vastin) [reply src/docs/setup/ec2.mdx]: Done

File: src/docs/setup/ec2.mdx (hunk 2 above)
Comment (vastin) [reply src/docs/setup/ec2.mdx]: Done

File: src/docs/setup/ec2.mdx (hunk 3 above)
Comment (vastin) [reply src/docs/setup/ec2.mdx]: Done

File: src/docs/setup/ec2.mdx (hunk 4 above)
Comment (vastin) [reply src/docs/setup/ec2.mdx:97]: Add link to SSM Distributor document for more installation options.

**Review by mxiamxia — APPROVED** [review]
Please add some screenshots later.

**Review by alolita — APPROVED** [review]
Will merge 4/29

**Pull request merged**